Example/ViewController.swift

```diff
@@ -311,7 +315,7 @@ class ViewController: UIViewController {
     func navigationService(route: Route, options: RouteOptions) -> NavigationService {
         let simulate = simulationButton.isSelected
         let mode: SimulationMode = simulate ? .always : .onPoorGPS
-        return MapboxNavigationService(route: route, routeOptions: options, directions: Settings.directions, simulating: mode)
+        return MapboxNavigationService(route: route, routeOptions: options, simulating: mode)
```

Comment:
`directions:` argument here is removed because it has default value of `Directions.shared`, which is the same as `Settings.directions`. This maintains the existing logic, but I see there are 2 ways:
(1) either we get rid of `Settings` at all since it is used only once and does nothing except introducing a synonym to existing singleton, or
(2) we preserve `Settings` for the future use, but in this case we should also preserve it in this line.

Reply:
removed `Settings` file